### ISSUE TITLE
Inject code snippet depend of mode

### ISSUE THREAD

**Issue by PixxxeL** [altrusl/vite-plugin-html-injection#2]
Are you consider an opportunity of inject code snippet depend of mode? It may be attribute `onBuild`, for example:

```javascript
        {
          name: "Google analytics",
          path: "./src/injections/ga.html",
          type: "raw",
          injectTo: "body",
          onBuild: true,
        }
```
If it `true` - code snippet inject only on Vite build or if `false` - inject always.

If you don't have such possibility, is may i make it and offer Pull Request?

**Comment by altrusl** [comment]
That seems to be useful 
I am going to add a more general property - `buildModes: dev | build | both`

**Comment by altrusl** [comment]
Implemented in 1.4.2 version via `buildModes` parameter (dev | prod | both)
